docs/next.config.mjs

```diff
@@ -22,29 +21,7 @@ const workspaceRoot = path.resolve(currentDirectory, '../');
 
 const withMdx = nextMdx({
   options: {
-    remarkPlugins: [
```

Comment:
@dav-is The plugin is breaking `og:image` on subpages

Reply:
> Is it still relevant? 🤔

Nope, it has been fixed in the meantime, this PR was demonstrating the bug.

The metadata cans till be a bit reorganized though I believe